parlai/agents/transformer/modules.py

```diff
@@ -230,51 +237,6 @@ def create_position_codes(n_pos, dim, out):
     out[:, 1::2] = torch.FloatTensor(np.cos(position_enc)).type_as(out)
 
 
-class TransformerResponseWrapper(nn.Module):
```

Comment:
I had to move these down so `TransformerEncoder` is declared before using it as a type. If we want to avoid moving these, I could just wrap the type in quotes `"TransformerEncoder"`, but that disables static checking of these.

Reply:
Ah the secret to that is to add at the top:

```python
from __future__ import annotations
```

This causes python to delay typechecking

Reply:
This ended up being resolved anyway by the breaking up of `modules.py`